business/services.go

```diff
@@ -34,7 +36,13 @@ type ServiceCriteria struct {
 }
 
 // GetServiceList returns a list of all services for a given criteria
-func (in *SvcService) GetServiceList(criteria ServiceCriteria) (*models.ServiceList, error) {
+func (in *SvcService) GetServiceList(ctx context.Context, criteria ServiceCriteria) (*models.ServiceList, error) {
+	var end observability.EndFunc
+	ctx, end = observability.StartSpan(ctx, "GetServiceList",
+		observability.Attribute("package", "business"),
```

Comment:
Same here, it could be good to pass the "criteria" in some way to the span to identify which criteria shows a bad latency.
Not a blocker, that can be done in a future iteration.